third-party/rocksdb/build.sh

```diff
@@ -23,7 +23,7 @@ echo
 
 cd $SOURCE_DIR
 
-compiler_flags="-fPIC -DPIC -DOPTDBG=1 -DROCKSDB_PLATFORM_POSIX -DROCKSDB_SUPPORT_THREAD_LOCAL -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_MALLOC_USABLE_SIZE -Wno-error=shadow  $EXTRA_CXXFLAGS"
+compiler_flags="-fPIC -DPIC -DROCKSDB_PLATFORM_POSIX -DROCKSDB_SUPPORT_THREAD_LOCAL -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_MALLOC_USABLE_SIZE -Wno-error=shadow  $EXTRA_CXXFLAGS"
```

Comment:
I don't know why this option was turned on, since it seems only making sense in the MSVC environment.

Good catch.